src/main/java/com/robinpowered/sdk/model/Identifier.java

```diff
@@ -72,6 +72,9 @@ public static Interface fromString(String value) {
     private final String value;
     private final DateTime createdAt;
 
+    // Mutable
+    private Integer deviceId;
```

Comment:
The property isn’t mutable, but it is nullable.

How do we want to handle adding it? If we change the constructor, that could be a BC break. We could alternatively overload the constructor to allow for setting the `deviceId`